install.sh

```diff
@@ -29,41 +29,43 @@ BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 echo "# Creating new environment for Macrel"
 mkdir -p envs
-conda create --yes -p $BASEDIR/envs/Macrel_env python=$PYTHON_VERSION
+conda create --yes -p $BASEDIR/envs/Macrel_env python=$PYTHON_VERSION ## Fixed 
+source activate $BASEDIR/envs/Macrel_env
 conda config --env --add channels defaults
 conda config --env --add channels bioconda
 conda config --env --add channels conda-forge
 
+echo "# Installing packages with MAMBA"
 if ! which mamba > /dev/null; then
     CONDA_INSTALL_CMD=conda
+    conda install -y \
+              ngless \
+              megahit \
+              paladin \
+              pandas \
+              scikit-learn \
+              atomicwrites \
+              tzlocal \
+              --quiet
 else
-    echo "# Installing packages with MAMBA"
-    CONDA_INSTALL_CMD=mamba
+    mamba install -y \
+              ngless \
+              megahit \
+              paladin \
+              pandas \
+              scikit-learn \
+              atomicwrites \
+              rpy2 \
+              tzlocal \
+              --quiet
 fi
-${CONDA_INSTALL_CMD} install -y \
```

Comment:
This seems like a pessimization, completely breaking DRY.

Reply:
corrected it